File: Simulator/src/IoTRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ASE_2017_Simulator
{
    class IoTRequest
    {
        public IoTStateChangedDTO stateChangedDTO { get; set; }
        public IoTNewSourceDTO newSourceDTO { get; set; }
    }
}

File: ClusterMasterService/src/main/java/ase/cm/EventProcessing/Models/IoTRequest.java
package ase.cm.EventProcessing.Models;

/**
 * Created by Tommi on 14.06.2017.
 */
public class IoTRequest {

    public IoTStateChangedDTO getStateChangedDTO() {
        return stateChangedDTO;
    }

    public void setStateChangedDTO(IoTStateChangedDTO stateChangedDTO) {
        this.stateChangedDTO = stateChangedDTO;
    }

    private IoTStateChangedDTO stateChangedDTO;
    private IoTNewSourceDTO newSourceDTO;




    public IoTNewSourceDTO getNewSourceDTO() {
        return newSourceDTO;
    }

    public void setNewSourceDTO(IoTNewSourceDTO newSourceDTO) {
        this.newSourceDTO = newSourceDTO;
    }
}
